packages/api-client/src/user/UserAPI.ts

```diff
@@ -40,6 +40,11 @@ import {
 import {RichInfo} from './RichInfo';
 import {RequestCancellationError} from './UserError';
 
+export type QualifiedUser = {
+  domain: string;
+  id: string;
+};
+
```

Comment:
Let's move this interface to `User.ts`, WDYT?

Reply:
Ok